packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [41.0.0]
+
+### Added
+
+- `TokenListController` now exports the type for `TokensChainsCache` ([#4821](https://github.com/MetaMask/core/pull/4821))
+
+### Changed
+
+- **BREAKING**: `CurrencyRateController` now accepts an array of native currencies as its polling input. ([#4852](https://github.com/MetaMask/core/pull/4852))
```

Comment:
This entry might need to provide some additional information on how it affects downstream consumers:

```suggestion
- **BREAKING**: The polling input accepted by `CurrencyRateController` is now an object with a `nativeCurrencies` property that is defined as a `string` array type ([#4852](https://github.com/MetaMask/core/pull/4852))
  - The `input` parameters of the controller's `_executePoll`, `_startPolling`, `onPollingComplete` methods now only accept this new polling input type.
  - `nativeCurrencies` replaces the `string`-type `nativeCurrency` property. 
```